Assets/Scripts/WorldEngine/Cultures/Preferences/CellCulturalPreference.cs

```diff
@@ -16,7 +16,7 @@ public class CellCulturalPreference : CulturalPreference
     private float _newValue;
 
 #if DEBUG
-    private long _lastUpdateDate = Manager.CurrentWorld.CurrentDate;
+    private long _lastUpdateDate = Manager.CurrentWorld == null ? 0 : Manager.CurrentWorld.CurrentDate;
```

Comment:
you can also do this:
_lastUpdateDate = Manager.CurrentWorld?.CurrentDate ?? 0;

from: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator